pool/src/storage.rs

```diff
@@ -359,6 +367,51 @@ pub fn has_res(e: &Env, asset: &Address) -> bool {
     e.storage().persistent().has(&key)
 }
 
+/// Fetch a queued reserve set
+///
+/// ### Arguments
+/// * `asset` - The contract address of the asset
+///
+/// ### Panics
+/// If the reserve set has not been queued
+pub fn get_queued_reserve_set(e: &Env, asset: &Address) -> QueuedReserveInit {
+    let key = PoolDataKey::ResInit(asset.clone());
+    e.storage()
+        .persistent()
```

Comment:
can this be temporary? No need for these to be kept alive longer than 7 days anyway.

Reply:
I still think we should store them for 30 days, but the temporary will be cheaper and more efficient for the blockchain